tests/get_kv_table_nodeos_tests.cpp

```diff
@@ -33,10 +33,10 @@ using namespace eosio::chain;
 using namespace eosio::testing;
 using namespace fc;
 
```

Comment:
Why not this file in unittests directory? I thought test directory was for integration testing and they were all python files.